CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
+### Added
+
+- Helper Method to add a restart_policy to PodBuilder([#565])
+
```

Comment:
Thank you! Looks good, just a minor fix otherwise the md linter wont be happy :)
```suggestion
## [Unreleased]

### Added

- Helper Method to add a restart_policy to PodBuilder([#565]).

[#565]: https://github.com/stackabletech/operator-rs/pull/565

```